pkg/pipelines/triggers/templates.go

```diff
@@ -98,13 +95,12 @@ func CreateCDPushTemplate(ns, saName string) triggersv1.TriggerTemplate {
 // CreateCIDryRunTemplate returns TriggerTemplate for CI Dry Try
 func CreateCIDryRunTemplate(ns, saName string) triggersv1.TriggerTemplate {
 	return triggersv1.TriggerTemplate{
-		TypeMeta: triggerTemplateTypeMeta,
-		ObjectMeta: meta.ObjectMeta(meta.NamespacedName(ns, "ci-dryrun-from-pr-template"),
-			statusTrackerAnnotations("ci-dryrun-from-pr-pipeline", "Stage CI Dry Run")),
```

Comment:
You lost this fix.   I think you copied you files on top of these files.  So, the fixes that are done before yours are lost.  Please go back to check the diffs to make sure the previous fixes are preserved.

Reply:
sure, on it